puppet/zulip/manifests/dockervoyager.pp

```diff
@@ -0,0 +1,41 @@
+# This class includes all the modules you need to install/run an Zulip installation
+# in a single container (without the database, memcached, redis services)
+# The database, memcached, redis services need to be run in seperate containers.
+# Through this split of services, it is easier to scale the services to the needs.
+class zulip::dockervoyager {
+  include zulip::base
+  # zulip::apt_repository must come after zulip::base
+  include zulip::apt_repository
+  include zulip::app_frontend
+
+  $appdb_packages = [# Needed to run process_fts_updates
+    "python-psycopg2",
```

Comment:
I think this may no longer be needed?  I think `process_fts_updates` knows how to find the Zulip virtualenv.